main.md

```diff
@@ -572,7 +572,76 @@ Cache-Control: no-cache, no-store
 }
 ```
 
-### Token Content {#token_content}
+### Enriched authorization details in Token Response
+
+There are use cases where the authorization server enriches the data in an authorization details object. For example, a client may ask for access to 
+account information but leave the decision about the accounts it will be able to access to the user. The user would select the sub set of accounts she 
+wants the client to entitle to access in the course of the authorization process. In order to allow the client to determine the accounts it is 
+entitled to access, the authorization server will add this information to the respective authorization details object. 
+
+As an example, the requested authorization detail parameter could look like this:
+
+```JSON
+"authorization_details": [
+   {
+      "type": "account_information",
+      "access": {
+         "accounts": [],
+         "balances": [],
+         "transactions": []
```

Comment:
This example implies that the client needs to make placeholders for these fields for the AS to fill in, but I don't think that's the intent. Can't the AS just add new fields to its response however it wants?

Reply:
This is a good question and I think the answer is really that it depends on how the type is defined - some might have placeholders of sorts and some might be more open ended towards the AS adding/modifying whatever it needs to. Or no? 

Regardless, some additional guidance is probably needed.

Reply:
I'm sure some types will have a fill-in approach, or post a suggested value and have it filled in with the actual value, etc. The section as written assumes that a blank is sent and the value is returned, though. I think it's a good pattern, but I don't think is the only pattern we're talking about.

Reply:
In this case, the client indicates its intention to access some accounts using this notation. It feels ok to me since the same syntax can be used to request access as determined by the user or pre-define some accounts the client already knows. 

I borrowed it from NextGenPSD2/Berlin Group.

Reply:
It makes sense that it's an allowed pattern, and maybe one that we call out with this very example, but I don't want to imply (or worse, require) that it's the only pattern for "enhancement".